src/modules/awaitAnimations.ts

```diff
@@ -58,7 +58,7 @@ export async function awaitAnimations(
 	await Promise.all(awaitedAnimations);
 }
 
-function awaitAnimationsOnElement(element: Element): Promise<void> | false {
+function awaitAnimationsOnElement(element: HTMLElement): Promise<void> | false {
```

Comment:
Changed to `HTMLElement` to allow strict typing of event handler durther down the line